client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -39,10 +42,12 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
       initialValues={{
         email: '',
         password: '',
-        username: '',
+        firstName: '',
+        lastName: '',
       }}
       validationSchema={Yup.object().shape({
-        username: Yup.string().required('Username is required').max(40, 'Username is too long'),
+        firstName: Yup.string().required('First Name is required').max(40, 'Username is too long'),
+        lastName: Yup.string().required('Last Name is required').max(40, 'Username is too long'),
```

Comment:
If the firstName and lastName exceed 40 characters the error we show is `Username is too long`. This should be changed to reflect the correct properties for better error messages

Reply:
Oops, I change it now.